apps/api/src/domain/models/vocabulary-list/queries/vocabulary-list-queries-fetch-many.e2e.spec.ts

```diff
@@ -202,14 +147,50 @@ describe.skip(`when querying for a vocabulary list: fetch many`, () => {
 
     let vocabularyListQueryRepository: IVocabularyListQueryRepository;
 
-    let seedVocabularyLists: (eventHistory: ICoscradEvent[]) => Promise<void>;
+    const setItUp = async (testUserWithGroups?: CoscradUserWithGroups) => {
+        // TODO avoid using `setUpIntegrationTest` here
+        ({ app, testRepositoryProvider, databaseProvider } = await setUpIntegrationTest(
+            {
+                ARANGO_DB_NAME: testDatabaseName,
+            },
+            {
+                testUserWithGroups,
+            }
+            // no authenticated user
+        ));
+
+        connectionProvider = app.get(ArangoConnectionProvider);
+
+        vocabularyListQueryRepository = new ArangoVocabularyListQueryRepository(
+            connectionProvider,
+            new ConsoleCoscradCliLogger()
+        );
+    };
+
+    const seedInitialState = async () => {
+        await vocabularyListQueryRepository.createMany([
+            // available to public user
+            publishedVocabularyList.clone({
+                entries: [buildEntryForTerm(publishedTerm)],
+            }),
+            // not available
+            privateVocabularyList.clone({
+                entries: [buildEntryForTerm(publishedTerm)],
+            }),
+            // not available
+            unpublishedVocabularyListOrdinaryUserCanAccess.clone({
+                entries: [buildEntryForTerm(publishedTerm)],
+            }),
+        ]);
+    };
 
     beforeEach(async () => {
         await testRepositoryProvider.testSetup();
 
-        await databaseProvider.getDatabaseForCollection('term__VIEWS').clear();
+        // TODO remove all such calls across all tests
```

Comment:
Let's do this.

Reply:
Ok but on the next branch. I'll end up touching too many files. I don't want to obscure the current work.